app/helpers/date_format_helper.rb

```diff
@@ -0,0 +1,26 @@
+module DateFormatHelper
+  def format_published_at(params)
+    if date_fields_present?(params)
+      begin
+        format_date(params)
+      rescue ArgumentError, RangeError => e
+        Rails.logger.info "Rescued: #{e.inspect}"
+        nil
+      end
+    end
+  end
+
+private
+
+  def date_fields_present?(params)
+    params.values.all?(&:present?)
```

Comment:
Hi, 

Only the `published_at` params are passed in so the method is doing what it says, but totally agree the naming isn't great. Have instead renamed `params` to be `date_fields` so that it looks less like all of the form params are being passed into the method.